diff --git a/modules/contrib/sharethis/js/sharethis.form.js b/modules/contrib/sharethis/js/sharethis.form.js
index 64aebd0..8b15b21 100644
--- a/modules/contrib/sharethis/js/sharethis.form.js
+++ b/modules/contrib/sharethis/js/sharethis.form.js
@@ -157,7 +157,7 @@
 
   Drupal.behaviors.shareThisForm = {
     attach: function (context) {
-      stlib_picker.setupPicker(jQuery('#myPicker'), drupalSettings.sharethis.service_string_markup, drupal_st.serviceCallback);
+      stlib_picker.setupPicker(jQuery('#myPicker', context), drupalSettings.sharethis.service_string_markup, drupal_st.serviceCallback);
       drupal_st.addEvents();
       drupal_st.setupServiceText();
     }
